client/inventory.go

```diff
@@ -1,43 +0,0 @@
-// Copyright 2016 Mender Software AS
-//
-//    Licensed under the Apache License, Version 2.0 (the "License");
-//    you may not use this file except in compliance with the License.
-//    You may obtain a copy of the License at
-//
-//        http://www.apache.org/licenses/LICENSE-2.0
-//
-//    Unless required by applicable law or agreed to in writing, software
-//    distributed under the License is distributed on an "AS IS" BASIS,
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//    See the License for the specific language governing permissions and
-//    limitations under the License.
-package client
-
-type InventoryAttribute struct {
```

Comment:
https://github.com/mendersoftware/inventory/blob/master/docs/devices_api.yml#L70 in case you don't believe me